### PR TITLE
APIGOV-27643 - add documentation for embedded azure creation via CLI

### PR DESCRIPTION
**Pull request by sbolosan**
Create setup page for building an embedded Azure agent via the CLIL
Deploy preview - http://pr-442.amplify-central.opendocs-builder.pcloud.axway.int/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | f69a86f |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-442.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    * There is a graphical environment (optional)
+* An Amplify platform user account that has the **Platform Administrator** and **Central Admin** roles
+* An Amplify platform service account to run the configuration in headless mode (optional)
+
+## Azure prerequisites
+
+* An Azure Service principal
+* An Azure Event Hub
+
+## Agent runner machine pre-requisites
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:45]: Agent runner machine?

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+   * **Resource Group Name** is the representation of group related resources for your application
+   * **API Management Service Name** refers to both the service and the corresponding Azure resource
+6. Azure Traceability Agent Configuration Setup options:
+   * **Event Hub Namespace** is the management container for one of multiple Event Hub instances
+   * **Event Hub Name** associated with the subscription
+   * **Policy Name** associated with the Event Hub
+   * **Policy Key** associated with the Policy Name
+   * **Consuer Group** associated with the Event Hub consumer group
+7. Traceability module connectivity:
+   * Traceability Agent protocol (Lumberjack (tcp) by default recommended for production environment or HTTPs recommended for testing purpose), select between `Lumberjack`, or `HTTPS`
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:153]: Traceability module connectivity?

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md (hunk 1 above)
Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:45]: oof... left overs.

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+Once you have answered all questions, the Embedded agent will be created. The process will securely store the authentication data and validate it by connecting to your Azure APIM. If set to discover the Azure APIM resources upon installation, the Embedded agent will immediately discover your resources and show them in the Service Registry.
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:155]: Immediately, if that options is selected, right?

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md (hunk 2 above)
Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:153]: left overs... fixing

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md (hunk 3 above)
Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:155]: yes, imediately

Comment (jcollins-axway) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:155]: oh, I see, in the cli we just run it right away every time, not just when an options is selected. got confused with ui for a tic

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 4]
```diff
@@ -42,8 +42,6 @@ Axway Central CLI and Amplify platform connectivity are required to configure th
 * An Azure Service principal
 * An Azure Event Hub
 
-## Agent runner machine pre-requisites
-
 The agents must have access to:
 
 * The platform URLs described in [Administer network traffic](/docs/connect_manage_environ/connected_agent_common_reference/network_traffic/) either directly or via a proxy
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:47]: The agents don't need access to anything...they have it as an embedded agent

Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:47]: my bad, i thought i wacked this in my last commit

**Review by jcollins-axway — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
appoved

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+* It is recommended that you have access to the Azure CLI command line to run the necessary setup commands
+
+## Objectives
+
+Learn how to quickly configure, install, and run your Embedded  agents with basic configuration using Axway Central CLI.
+
+Axway Central CLI and Amplify platform connectivity are required to configure the agent.
+
+1. Configure the agent from any machine that has access to the Amplify platform (<https://platform.axway.com>) and a graphical environment (optional).
+2. Once the configuration is complete, the agent configuration must be copied to a Docker machine so that the Docker runner will find them.
```
Comment (lbadenhop) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:27]: I see two objectives, but I only see documentation for the first "Configuration" objective that includes Steps 1, 2 and 3. Did I miss something? Are there instructions for copying the agent configuration to the Docker machine?

Comment (sbolosan) [reply content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:27]: Nope you're right. There is no docker configuration.  Will remove

**Comment by lbadenhop** [comment]
Can you please check the references to Central CLI, Axway CLI and Axway Central CLI? They are used interchangeably in some sections, but I'm not sure they should be since we do have both an Axway Central CLI and an Axway CLI.

File: content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+### Step 2: Identify yourself to Amplify platform with Axway CLI
+
+There are two ways to authenticate with Axway CLI:
+
+* Default mode: with an administrator username (email address) / password via a browser
+* Headlessmode: with a platform service account and a username / password via a prompt
+
+#### Default mode with browser authentication
+
+Run the following command to use Central CLI to log in with your Amplify platform credentials:
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_azure_gateway/deploy-embedded-agents.md:65]: This should be Axway Central CLI

**Comment by sbolosan** [comment]
Thank you for the inconsistency check... I commented on one that should be changed.  The other references to Axway CLI and Axway Central CLI are correct

**Pull request merged**